codility_training/lessons.lesson10.PrimeAndCompositeNumbers.MinPerimeterRectangle/NowonLee.js

```diff
@@ -0,0 +1,12 @@
+function solution(N) {
+    let l = []
```

Comment:
```suggestion
    const l = []
```
배열의 경우 const로 선언! 배열 자체가 변경되진 않기 때문에 const로 해주는게 맞습니다.